src/MapboxStyleParser.ts

```diff
@@ -682,12 +699,11 @@ export class MapboxStyleParser implements StyleParser<Omit<MbStyle, 'sources'>>
       });
     } else {
       // returns array of rules where one rule contains one symbolizer
-      layers.forEach(layer => {
+      layers.forEach((layer, layerIdx) => {
```

Comment:
Is layerIdx used somewhere? Should be marked by the linter otherwise.

Reply:
good catch. fixed.